### PR TITLE
Fixed issue #17873: Remote Control `delete_language` fails to remove language

### PR DESCRIPTION
**Pull request by edgarrmondragon**
Fixed issue #17873:  Remote Control `delete_language` fails to remove language

### CODE REVIEW

File: application/helpers/remotecontrol/remotecontrol_handle.php [hunk 1]
```diff
@@ -828,7 +828,8 @@ public function delete_language($sSessionKey, $iSurveyID, $sLanguage)
                     return array('status' => 'Cannot remove base language');
                 }
                 $aLanguages = $oSurvey->getAdditionalLanguages();
-                unset($aLanguages[$sLanguage]);
+                $iLanguageKey = array_search($sLanguage, $aLanguages, true)
```
Comment (olleharstedt) [inline application/helpers/remotecontrol/remotecontrol_handle.php:831]: Missing semi-colon on this line

**Review by olleharstedt — CHANGES_REQUESTED** [review]

File: application/helpers/remotecontrol/remotecontrol_handle.php (hunk 1 above)
Comment (edgarrmondragon) [reply application/helpers/remotecontrol/remotecontrol_handle.php]: 🤦 thanks @olleharstedt! I fixed it.

**Comment by olleharstedt** [comment]
@Shnoulle You want to review this change before merge?

**Comment by Shnoulle** [comment]
? Seems OK, but not tested .

**Comment by edgarrmondragon** [comment]
@Shnoulle I assumed this method was already being tested somewhere in [tests/unit/helpers](https://github.com/LimeSurvey/LimeSurvey/tree/master/tests/unit/helpers) but you're right there are none. I might be able to add a test but probably not until the weekend.

**Comment by Shnoulle** [comment]
@edgarrmondragon : i just mean i didn't test it manually ;)

**Comment by c-schmitz** [comment]
@edgarrmondragon Adding a test would be great. We will wait with the merge until then. Thank you for your help!

**Review by c-schmitz — APPROVED** [review]

**Pull request merged**